source/PopNGo/PopNGo/PopNGo/wwwroot/js/viewEvents.js

```diff
@@ -10,6 +12,76 @@ function displayEvents(events) {
 
     events.forEach(event => {
         // Create elements for each event and append them to the container
+        const heart = new Image();
+        heart.alt = 'Favorite/Unfavorite Event';
+        heart.classList.add('heart');
+        heart.style.cursor = 'pointer'; //might want to add this to css if possible, but i dont think its necessary
```

Comment:
Possibly a class to help with the image's size on the page? Seems a bit big currently. Not required, I suppose.